jadx-gui/build.gradle

```diff
@@ -51,7 +51,7 @@ startScripts {
         def str = windowsScript.text
         str = str.replaceAll('java.exe', 'javaw.exe')
         str = str.replaceAll('"%JAVA_EXE%" %DEFAULT_JVM_OPTS%', 'start "jadx-gui" /B "%JAVA_EXE%" %DEFAULT_JVM_OPTS%')
-        str = str.replaceAll('set JAVA_EXE=%JAVA_HOME%/bin/javaw.exe', 'set JAVA_EXE="%JAVA_HOME%/bin/javaw.exe"')
+        str = str.replaceAll('set JAVA_EXE=%JAVA_HOME%/bin/javaw.exe', 'set JAVA_EXE=%JAVA_HOME%/bin/javaw.exe')
```

Comment:
Looks like we can remove this line completely. Also please apply this change also in `jadx-cli/build.gradle` .
And for reference I made these changes here: a051ce6cf4ec2f9a431366658eebbcc988d9d11d, sorry :(

Reply:
LOL you are right. I did not recognize that the line is now totally useless.